docs/source/api/plugin/usage-reporting.md

```diff
@@ -111,27 +111,37 @@ The only properties of the reported error you can modify are its `message` and i
+Returning `false` (or `0`) for some or all operations can improve your server's performance, because calculating complete traces can introduce significant overhead. This is especially true for a federated graph, because traces are transmitted from the subgraph to the Gateway in-band inside the actual GraphQL response.
+
+**If the function returns a positive number:**
 
-Returning a positive number means that Apollo Server will track each field execution and send Apollo Studio statistics on how many times each field was executed and what the per-field performance was. Apollo Server sends both a precise observed execution count and an estimated execution count. The former is calculated by counting each field execution as 1, and the latter is calculated by counting each field execution as the number returned from this hook, which can be thought of as a weight.
+* Apollo Server _does_ calculate per-field statistics for the associated operation, and it sends those statistics to Apollo Studio.
+* Apollo Server sends Studio both an _observed_ execution count and an _estimated total_ execution count for each field.
+* The _observed_ execution count is exactly how many times each field was resolved in the associated operation.
+* The _estimated total_ execution count is the _observed_ execution count, _multiplied by the number returned by this function_. You can think of this returned number as an "estimation multiplier".
 
-Passing a number `x` (which should be between 0 and 1 inclusive) for `fieldLevelInstrumentation` is equivalent to passing the function `async () => Math.random() < x ? 1/x : 0`.  For example, if you pass 0.01, then 99% of the time this function will return 0, and 1% of the time this function will return 100. So 99% of the time Apollo Server will not track field executions, and 1% of the time Apollo Server will track field executions and send them to Apollo Studio both as an exact observed count and as an "estimated" count which is 100 times higher.  Generally, the weights you return should be roughly the reciprocal of the probability that the function returns non-zero; however, you're welcome to craft a more sophisticated function, such as one that uses a higher probability for rarer operations and a lower probability for more common operations.
+The number you return from the function should be approximately the reciprocal of the probability that the function returns a non-zero number. For example, if the function returns `0` with a probability of `0.1`, then it should return `10` the rest of the time. However, you can craft a more sophisticated function, such as one that uses a higher probability for rarer operations and a lower probability for more common operations.
```

Comment:
I think maybe add "for this operation" to the end of the first sentence. The idea is that the probability might vary across operations but the probability used for any particular operation should be the reciprocal of what is returned.

(There are also non-random strategies you can use, like "return non-zero for the first instance of an operation in each minute and zero for all the rest, and that non-zero number is based on the number of times the operation ran last minute" fwiw.)